pkg/types/k8s/factory/configmap.go

```diff
@@ -0,0 +1,22 @@
+package factory
```

Comment:
is it necessary to have a `factory` package? it seems this would be alright to nest under `k8s`

Reply:
ill remove it in the interest of fewer packages.  i wasnt certain there wouldnt be some other needs but YAGNI